oauth2_http/javatests/com/google/auth/oauth2/IdentityPoolCredentialsTest.java

```diff
@@ -35,74 +35,46 @@
 import static com.google.auth.oauth2.MockExternalAccountCredentialsTransport.SERVICE_ACCOUNT_IMPERSONATION_URL;
 import static com.google.auth.oauth2.OAuth2Utils.JSON_FACTORY;
 import static org.junit.Assert.*;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
 
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.json.GenericJson;
 import com.google.api.client.util.Clock;
 import com.google.auth.TestUtils;
 import com.google.auth.http.HttpTransportFactory;
+import com.google.auth.mtls.X509Provider;
 import java.io.ByteArrayInputStream;
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import javax.annotation.Nullable;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.junit.runners.JUnit4;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /** Tests for {@link IdentityPoolCredentials}. */
-@RunWith(JUnit4.class)
+@RunWith(MockitoJUnitRunner.class)
 public class IdentityPoolCredentialsTest extends BaseSerializationTest {
 
   private static final String STS_URL = "https://sts.googleapis.com/v1/token";
 
-  private static final Map<String, Object> FILE_CREDENTIAL_SOURCE_MAP =
-      new HashMap<String, Object>() {
-        {
-          put("file", "file");
-        }
-      };
-
-  private static final IdentityPoolCredentialSource FILE_CREDENTIAL_SOURCE =
-      new IdentityPoolCredentialSource(FILE_CREDENTIAL_SOURCE_MAP);
-
-  private static final IdentityPoolCredentials FILE_SOURCED_CREDENTIAL =
```

Comment:
I moved these setups to helper functions because they are not needed for every tests.